Resources/public/js/views/tabs/ez-locationviewlocationstabview.js

```diff
@@ -141,6 +144,73 @@ YUI.add('ez-locationviewlocationstabview', function (Y) {
          */
         _enableSetMainLocationRadios: function () {
             this.get('container').all('.ez-main-location-radio').set('disabled', false);
+        },
+
+        /**
+         * Tap event handler on the `Remove selected` button. It fires the
+         * `removeLocations` event
+         *
+         * @method _removeSelectedLocations
+         * @protected
+         * @param {EventFacade} e
```

Comment:
Yep, you are right - it's not used. Should it be removed then even if it's kind of sign that it's event handler?